subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -1313,7 +1312,7 @@
    *
    * This method is not appropriate for long running or blocking callables.
    * If callable is long running or blocking use
-   * {@link #blocking(String, Callable, Executor) blocking} method.
+   * {@link #runInExecutor(String, Callable, Executor) blocking} method.
```

Comment:
update blocking to runInExecutor. same for other places

Reply:
I think here the word `blocking` does not need to be replaced.

Reply:
text will point to #blocking method but link for new method in parsed doc. It will be confusing since we can't remove blocking method